requirements.txt

```diff
@@ -1,7 +1,7 @@
 polygon-api-client
 alpha_vantage
 ibapi==9.81.1.post1
-yfinance
+yfinance==0.2.44
```

Comment:
this will not fix the issue, it needs to be updated in setup.py. there's also a 2.46 out now, checking if this issue is fixed